src/Command/InstallWebDirCommand.php

```diff
@@ -108,8 +181,75 @@ private function addFiles($webDir)
      */
     private function removeInstallPhp($webDir)
     {
-        if (file_exists($webDir.'/install.php')) {
-            $this->fs->remove($webDir.'/install.php');
+        if (!file_exists($webDir.'/install.php')) {
+            return;
         }
+
+        $this->fs->remove($webDir.'/install.php');
+        $this->io->text('Deleted the <comment>web/install.php</comment> file.');
+    }
+
+    /**
+     * Stores username and password in .env file in the project directory.
+     *
+     * @param InputInterface $input
+     * @param string         $projectDir
+     */
+    private function storeAppDevAccesskey(InputInterface $input, $projectDir)
+    {
+        $user = $input->getOption('user');
+        $password = $input->getOption('password');
+
+        if (null === $password && null === $user) {
+            return;
+        }
+
+        if (true === $input->getOption('no-dev') && (null !== $user || null !== $password)) {
+            throw new \InvalidArgumentException('Cannot set a password in no-dev mode!');
+        }
+
+        if ((null === $password && null !== $user) || (null === $user && null !== $password)) {
+            throw new \InvalidArgumentException('Must have username and password to set the access key.');
+        }
+
+        $accessKey = hash(
```

Comment:
Reply to https://github.com/contao/manager-bundle/pull/33#issuecomment-304280979 which should have been here but got sorted on the generic discussion below:
> We already do? We have basic authentication, the standard does not?

Let me put it another way, we have strong authentication (with bcrypt, salting etc.) in application for Backend and Frontend users, we have it for the manager itself (uses bcrypt as suggested by me here) but the dev entry point shall be protected by sha512 only?
Given that most people (IMO) will add an generic user named "admin" or "developer" this will open us up for rainbow table attacks.

I wonder why nobody else from @contao/developers has an opinion on this one.